tests/kernel/update_kernel.pm

```diff
@@ -290,9 +290,11 @@ sub prepare_kgraft {
     $src_name .= '-' . $$incident_klp_pkg{kflavor}
       unless $$incident_klp_pkg{kflavor} eq 'default';
 
+    zypper_call("mr -e kgraft-test-repo-0") if get_var('FLAVOR') =~ /-Updates-Staging/;
```

Comment:
Are you sure there will never be more than one staging repo?

Reply:
Yes, there will be always one repo per staging.

Reply:
OK